src/Game.js

```diff
@@ -14,87 +14,98 @@ function Game () {
 		])
 	)
 
-	const handleEnemyEndOfTurn = (playerName) => {
-		console.log('Your turn ' + playerName) // TODO: Remove these
-
+	function handleEnemyEndOfTurn (playerName) {
 		setWhoseTurn(playerName)
 	}
 
-	const handlePlayerHasLost = (playerName) => {
-		const winner =
-			playerName === PLAYER_ONE_NAME ? PLAYER_TWO_NAME : PLAYER_ONE_NAME
+	function handlePlayerHasLost (playerName) {
+		const winner = playerName === PLAYER_ONE_NAME ? PLAYER_TWO_NAME : PLAYER_ONE_NAME
 
-		console.log('You lost ' + playerName)
-		console.log('Congratulations ' + winner)
-		setGameOver(true)
+		setWhoWon(winner)
 	}
 
-	const handlePlayerShipPlacement = (playerName) => {
-		const whoseTurn =
-			playerName === PLAYER_ONE_NAME ? PLAYER_TWO_NAME : PLAYER_ONE_NAME
+	function handlePlayerShipPlacement (playerName) {
+		const whoseTurn = playerName === PLAYER_ONE_NAME ? PLAYER_TWO_NAME : PLAYER_ONE_NAME
 		const shipsOnBoardCopy = new Map(shipsOnBoard)
 
 		shipsOnBoardCopy.set(playerName, true)
 		setWhoseTurn(whoseTurn)
 		setShipsOnBoard(shipsOnBoardCopy)
 	}
 
-	return <div className='game'>{renderPlayers()}</div>
+	function renderAPlayer (name, hideBoard, showShipOnBoard, unclickableBoard = false) {
```

Comment:
In my opinion here, especially after scrolling down, having this in a function makes it so much harder to know what exactly you're passing. I would highly suggest not doing this - especially with the amount of boolean parameters that you're passing in